lotus/models/lm.py

```diff
@@ -65,14 +66,19 @@ def __call__(self, messages: list[list[dict[str, str]]], **kwargs: dict[str, Any
             [self._get_top_choice_logprobs(resp) for resp in all_responses] if all_kwargs.get("logprobs") else None
         )
 
+        self.print_total_usage()
+        self.reset_stats()
```

Comment:
I don't think you should call `reset_stats` since the intent of total usage is to track usage for the LM's lifetime.